rdf/rdf12/rdf-turtle/eval/turtle-star-eval-annotation-6.ttl

```diff
@@ -0,0 +1,3 @@
+PREFIX : <http://example/>
+
+:s :p :o {| :i | :r :z |} .
```

Comment:
Note that this syntactic form of "naming" the annotation is [problematic](https://lists.w3.org/Archives/Public/public-rdf-star-wg/2024Jan/0097.html) (it won't work in SPARQL, due to the [AlternativePath operator](https://www.w3.org/TR/sparql11-query/#pp-language)).

Reply:
See https://github.com/w3c/rdf-star-wg/issues/116.

Reply:
Syntax not up to date.